src/eos-autoupdater.c

```diff
@@ -66,6 +66,7 @@ static const char *UPDATE_STAMP_NAME = "eos-updater-stamp";
 static const char *CONFIG_FILE_PATH = SYSCONFDIR "/" PACKAGE "/eos-autoupdater.conf";
 static const char *OLD_CONFIG_FILE_PATH = SYSCONFDIR "/eos-updater.conf";
 static const char *STATIC_CONFIG_FILE_PATH = PKGDATADIR "/eos-autoupdater.conf";
+static const char *VAR_CONFIG_FILE_PATH = LOCALSTATEDIR "/local/" PACKAGE "/eos-autoupdater.conf";
```

Comment:
There is great precedent for checking both /usr/local and /usr - eg XDG_DATA_DIRS. Assuming /usr is read-only and using a non-standard /var path instead would be introducing dependencies on the ostree implementation detail. 

The assumption that the image builder is relying on, that /usr/local will end up in /var/local is the ostree-specific thing - but in fact the image builder also creates the symlink so the hook could also act on /usr/local and the file would end up in the right place.

But in the end... neither the updater nor the image builder can be expected to be that independent of ostree... so it doesn't matter all that much as long as it works!